feature-libs/order/root/tokens/context.ts

```diff
@@ -13,3 +13,8 @@ export const OrderDetailsOrderEntriesContextToken = new InjectionToken(
 export const OrderConfirmationOrderEntriesContextToken = new InjectionToken(
   'OrderConfirmationOrderEntriesContext'
 );
+
+export const MYACCOUNT_ORDER_ENHANCED_UI = new InjectionToken<boolean>(
```

Comment:
[MEDIUM]
Of course the final naming of the suffix qualifier (e.g. `Extended` vs `EnhancedUi` vs .....) is to be decided. But regardless of the decision, IMHO the name of the feature should go after the full qualifier:
`MYACCOUNT_ORDER_ENHANCED_UI` vs `MYACCOUNT_ENHANCED_UI_ORDER`.
Thanks to that we will have a common prefix and only the ending will change among other features, e.g.:
```
MYACCOUNT_ENHANCED_UI_ORDER
MYACCOUNT_ENHANCED_UI_CHANGE_PASSWORD
MYACCOUNT_ENHANCED_UI_USER_PROFILE
...
```

Reply:
done